src/NuGet.Clients/NuGet.CommandLine/Commands/VerifyCommand.cs

```diff
@@ -39,12 +40,23 @@ public override Task ExecuteCommandAsync()
                 throw new ArgumentNullException(nameof(PackagePath));
             }
 
+            if (ConfigFile != null)
```

Comment:
the base class command already did this and created a packageSourceProvider, I think what you need during verification is Lists of PackageSources or just TrustedSource, How about passing those sources to verifyArgs